f8a_report/v2/report_generator.py

```diff
@@ -199,6 +189,24 @@ def analyse_stack(self, stacks_data, report_template) -> dict:
         logger.info("Stacks Analyse Completed.")
         return report_template
 
+    def collate_vulnerabilites(self, stack_info_template, package):
```

Comment:
Isnt this the same code moved to a different function? Is it a refactor task or a bug fix?

Reply:
Its refactor + Bug Fix 
Actually Yesterday's Invocation caused above mentioned KeyError on Stage. 
This fix + refactor is related to that.

Reply:
Yesterday's report was generator successfully but was missing CVE info due to KeyError occurred. because of Key name change (CVE -> cve_id ) in v2 format.

Looking at code, I felt there is a need for refactoring at this point along with Bug Fix.